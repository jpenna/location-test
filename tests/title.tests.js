/* eslint import/no-extraneous-dependencies: "off" */
/*  eslint import/no-unresolved: "off" */

import React from 'react';
import { render } from 'enzyme';

import Title from 'src/components/title';

describe('Title', () => {
  it('should have H1 title', () => {
    const component = render(<Title />);
    expect(component.find('h1').text()).toEqual('GeoLocation Test');
  });
});
