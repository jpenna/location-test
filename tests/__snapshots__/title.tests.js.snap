// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Title /> should render correctly 1`] = `
<div
  className="has-text-centered"
>
  <h1
    className="title is-1"
  >
    GeoLocation Test
  </h1>
  <h2
    className="subtitle has-margin-bottom-10"
  >
    Find you and a website
  </h2>
</div>
`;
